### DISCUSSION TITLE
DNS cache for output plugin

### DISCUSSION THREAD

**Discussion by jdomag** [fluent/fluentd#3865] [Q&A]
Hi, 
We configured our fluentd in k8s to send the logs to graylog by using the image:
 `fluent/fluentd-kubernetes-daemonset: v1.15.1-debian-graylog-amd64-1.0`
As far as I know it uses [gelf plugin](https://github.com/hotschedules/fluent-plugin-gelf-hs).
We noticed that our coreDNS in k8s is flooded with the DNS requests from fluentd (it asks about graylog host) - looks like fluentd is not caching the response and with every batch that needs to be send to graylog, the DNS request is generated. 
How to enable the dns cache for fluentd? Is it something that should be configured on fluentd level or on the output plugin level?

**Comment by shovelend** [comment]
Any news on this one? We're affected by this issue too.

**Comment by daipom** [comment]
Thanks for your report.
I will check it soon.

**Reply by shovelend** [reply]
It'd be great if you could check it for us @daipom , thank you.
